plugin/src/leiningen/cljsbuild/subproject.clj

```diff
@@ -71,13 +72,18 @@
          (map (fn [[k v]] (vec (cons k v)))))))
 
 (defn make-subproject [project crossover-path builds]
-  (with-meta
-    (merge
+  (let [deps (classpath/merge-versions-from-managed-coords
```

Comment:
Can you check if this var exists before calling it? If it does then use it, if not then fall back to the old behaviour?